plugin/mlsdbprovider.cpp

```diff
@@ -316,8 +327,13 @@ void MlsdbProvider::tryFetchOnlinePosition()
             connect(m_mlsdbOnlineLocator, &MlsdbOnlineLocator::error,
                     this, &MlsdbProvider::onlineLocationError);
         }
-        if (m_mlsdbOnlineLocator->findLocation(cellIds)) {
-            return;
+        const QPair<QDateTime, QVariantMap> query = m_mlsdbOnlineLocator->buildLocationQuery(
+                cellIds, m_previousQuery);
+        if (!query.first.isNull() && !query.second.isEmpty()) {
```

Comment:
Silently regards the error messages from findLocation as the check is done now in two places.

Reply:
I will remove this check here and add the timestamp check to the findLocation